libraries/botbuilder-dialogs-adaptive/src/actions/httpRequest.ts

```diff
@@ -8,21 +8,22 @@
 import fetch from 'node-fetch';
 import { Response, Headers } from 'node-fetch';
 import {
-    ValueExpression,
-    StringExpression,
     BoolExpression,
-    EnumExpression,
     BoolExpressionConverter,
+    EnumExpression,
     EnumExpressionConverter,
+    Expression,
+    StringExpression,
     StringExpressionConverter,
+    ValueExpression,
     ValueExpressionConverter,
 } from 'adaptive-expressions';
 import { Activity } from 'botbuilder-core';
 import { Converter, Converters, DialogTurnResult, DialogContext, Dialog } from 'botbuilder-dialogs';
 import { replaceJsonRecursively } from '../jsonExtensions';
 
-class HttpHeadersConverter implements Converter<object, { [key: string]: StringExpression }> {
-    public convert(value: object): { [key: string]: StringExpression } {
+class HttpHeadersConverter implements Converter<Record<string, string | Expression>, Record<string, StringExpression>> {
```

Comment:
Since the `Converter` type parameters are slightly complex and duplicated in the `convert` signature, consider extracting them to type aliases such as:

```typescript
type Input = Record<string, string | Expression>
type Output = Record<string, StringExpression>

class HttpHeadersConverter implements Converter<Input, Output> {
  public convert(value: Input): Output { ... }
}
```